Check broken links (#4)
christinerose/ocaml.org: data/planet/ocamlorg/opam-in-xenserver.md

*This is a guest post from an OPAM user about how they use it.  If you would like to post
 about your own use, [please let us know](https://github.com/ocaml/platform-blog/issues).*

> [XenServer](http://www.xenserver.org/ - [404 Not Found]) uses the

Reviewer: ```suggestion
[Citrix Hypervisor (formerly known as XenServer)](https://www.citrix.com/products/citrix-hypervisor/) uses the
```

which are
developed independently and versioned separately. It's easy for us to
share code with other open-source projects like

> [Mirage](http://www.openmirage.org/ - [1 Client error: Timeout was reached]), however

Reviewer: ```suggestion
[Mirage](https://mirage.io/), however
```

The Xapi toolstack is built from a large set of libraries, independently
versioned and released, many of them shared with other projects

> (such as [Mirage](http://www.openmirage.org/ - [1 Client error: Timeout was reached])). The libraries are

Reviewer: ```suggestion
(such as [Mirage][(http://www.openmirage.org/ - [1 Client error: Timeout was reached]](https://mirage.io/)). The libraries are
```
